schemas/cslc_s1.yaml

```
@@ -0,0 +1,74 @@
+runconfig:
+    name: str()
+
+    groups:
+        pge_name_group:
+           pge_name: enum('CSLC_S1_PGE')
+
+        input_file_group:
+           # Required. List of SAFE files (min=1)
+           safe_file_path: list(str(), min=1)
+           # Required. List of orbit (EOF) files
+           orbit_file_path: list(str(), min=1)
+           # List of (unique) burst ID to process
+           burst_id: list(str(), min=0, max=27, required=False)
+           # Reference burst options
+           reference_burst: include('reference_burst_options', required=False)
+
+        dynamic_ancillary_file_group:
+           # Digital Elevation Model.
+           dem_file: include(str(), required=False)
+
+        product_path_group:
+           # Directory where PGE will place results
+           product_path: str()
+           # Directory where SAS can write temporary data
+           scratch_path: str()
+           # Intermediate file name. SAS writes the output to this file.
+           # PGE may rename the product according to file naming convention
+           sas_output_file: str()
+
+        primary_executable:
+           product_type: enum('CSLC_S1')
+
+        # This section includes parameters to tweak the workflow
+        processing:
+           # Polarization to process. If not set, process all polarizations for that burst
+           polarization: any(list('HH', 'HV', 'VH', 'VV'), min=1, max=4), str(min=2, max=2), required=False)
+
+           # Options to run rdr2geo
+           rdr2geo: include('rdr2geo_options', required=False)
+
+           # Options to run geo2rdr
+           geo2rdr: include('rdr2geo_geo2rdr_shared_options', required=False)
+
+           # Options to run resample
+           resample: include('resample_options', required=False)
+
+
+# Group of processing options
+reference_burst_options:
+    # Required. Flag indicating reference burst
+    is_reference: bool()
```

```suggestion
    # Required. If True, rdr2geo will be performed on the SAFE files and burst IDs above
    is_reference: bool()
```
Documentation updated to account for a case where: for a given ROI made up of a collection of burst IDs, on the day chosen as reference, burst IDs span 2 swaths. If reference days are chosen to avoid the condition I described, then this comment maybe ignored..

---

Can you elaborate more what you mean?

---

I'm trying to account for the following scenario:

Reference ROI spans more than 1 swath: ROI includes IDs: 1, 2, 3. IDs 1 and 2 are in swath N and  and ID 3 is in swath N+1. For the YAML capture this, `safe_file_path` will have more than item, (swath N and swath N+1) and `is_reference` is `True`.

---

@LiangJYu I think your suggestion seems a bit too detail for an end user who may want to only deal with the run config. How about ```Flag indicating reference burst. If True the input zip file is a reference date```. What Virginia has  also is clear to me.

---

Sounds good to me.